app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala

```diff
@@ -57,34 +56,21 @@ trait AdjustedGroupInterestValidator extends BaseValidation {
     }
   }
 
-  private def validateGroupEBITDA(implicit path: JsPath): ValidationResult[BigDecimal] = {
-    if (adjustedGroupInterestModel.groupEBITDA % 0.01 != 0) {
-      GroupEBITDADecimalError(adjustedGroupInterestModel.groupEBITDA).invalidNec
-    } else {
-      adjustedGroupInterestModel.groupEBITDA.validNec
+  private def validateGroupEBITDA(implicit path: JsPath): ValidationResult[Option[BigDecimal]] = {
+    adjustedGroupInterestModel.groupEBITDA match {
+      case Some(ebitda) if ebitda % 0.01 != 0 => GroupEBITDADecimalError(ebitda).invalidNec
+      case _ => adjustedGroupInterestModel.groupEBITDA.validNec
     }
   }
 
   private def validateGroupRatioCalculation(implicit path: JsPath): ValidationResult[BigDecimal] = {
     (adjustedGroupInterestModel.qngie, adjustedGroupInterestModel.groupEBITDA, adjustedGroupInterestModel.groupRatio) match {
-      case (_, groupEBITDA, groupRatio) if groupEBITDA <= 0 && groupRatio == 100 =>
-        adjustedGroupInterestModel.groupRatio.validNec
-      case (_, groupEBITDA, groupRatio) if groupEBITDA <= 0 && groupRatio != 100 =>
+      case (_, Some(groupEBITDA), groupRatio) if groupEBITDA <= 0 && groupRatio != 100 =>
         NegativeOrZeroGroupEBITDAError(groupRatio).invalidNec
       case _ => adjustedGroupInterestModel.groupRatio.validNec
     }
   }
 
-  private def groupRatioCalculationDoesntMatch(qngie: BigDecimal, groupEBITDA: BigDecimal, groupRatio: BigDecimal): Boolean = {
```

Comment:
Has this been deleted on purpose?

Reply:
Yeah, it wasn't used. We removed its use in a previous development but we mustn't have removed the method itself